docs/tutorials/protect_against_regen.rst

```diff
@@ -0,0 +1,48 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _protect_against_regen:
+
+*************************************************************************
+Configuring Your Module to Protect Against Regeneration Voltage Spikes
+*************************************************************************
+
+What are Regenerative Voltage Spikes
+=======================================
+Regeneration voltage occurs when the commanded module voltage is less than the module's back-EMF voltage. When this occurs, the motor's kinetic 
+energy is converted directly to electrical energy, and produces a current flowing out of the motor and into the power source.
+
+The commanded module voltage may be less than the back-EMF voltage in cases where the module is slowing down or braking. The faster that the commanded voltage 
+changes (for example, jumping from a 20V to a 0V command), the greater the magnitude of the regeneration current.
+
+How Can Regeneration Affect Your Module
+=========================================
+If the module's power source can absorb the regenerated current without allowing the voltage to spike, regeneration protection isn't needed. This is often true with a battery as the power source.
+When connected to a power source that cannot absorb this regenerated current, which is generally the case with benchtop power supplies, the output voltage of the source will often increase during regeneration. 
+These spikes can result in voltages above the module's and your supply's maximum rated operating voltage. When this happens, you can permanently damage the module, the power supply, or both.
+
+Protecting Against Dangerous Regeneration Voltage Spikes
+===========================================================
+Regenerative voltage spikes can pose a threat to your modules and power supply if not properly controlled. As such, all Vertiq modules can limit their regeneration current to limit the magnitude of voltage spikes on power supplies. 
+Vertiq modules have two main parameters to protect against dangerous regeneration voltage spikes. They are available through IQ Control Center's advanced tab as *Volts Limit* and *Volts Limit Starting Voltage*.
+
+For example, by default on Vertiq 81-XX Gen 2 family modules (rated to 14S), you will see:
```

Comment:
I think you should refrain from using concrete examples.  If the specs change, you'll have to come back and change this example, which we almost certainly won't remember to do.  

Instead, you can just say "For example, for our modules rated to 14S, you would see:"